### ISSUE TITLE
Setting config with spaces creates config sections

### ISSUE THREAD

**Issue by mskymoore** [clueboard/milc#10]
```
milc_app config general.some_config="Some string with spaces"
```
doing this creates sections in the config file for every word after the first space:
```
[general]
some_config = Some

[string]

[with]

[spaces]
```

**Comment by skullydazed** [comment]
Nice catch! Not sure what's going on here yet but it's something we need to fix.

**Comment by skullydazed** [comment]
This should be fixed in 1.0.7.

**Comment by mskymoore** [comment]
@skullydazed Verified.  Nice!
